patches/build_qemu.diff

```diff
@@ -0,0 +1,147 @@
+--- qemu_mode/build_qemu_support.sh	2019-01-25 19:26:49.314879371 -0700
```

Comment:
Are these invasive changes to the build process necessary? The diff reads like a replacement of the entire build script. Is it a case of the build script continuing to evolve in upstream AFL? Let's try to have minimal necessary changes here.